datasync/datasync.go

```diff
@@ -367,3 +367,49 @@ func (d *Datasync) GetDatasyncTags(ctx context.Context, tArn string) ([]*datasyn
 
 	return out.Tags, err
 }
+
+// StartTaskExecution starts the execution and returns the taskexecution ARN
+func (d *Datasync) StartTaskExecution(ctx context.Context, taskArn string) (*datasync.StartTaskExecutionOutput, error) {
+	if taskArn == "" {
+		return nil, apierror.New(apierror.ErrBadRequest, "invalid input", nil)
+	}
+
+	log.Info("starting datasync task executions")
+
+	filters := &datasync.StartTaskExecutionInput{TaskArn: &taskArn}
```

Comment:
we usually use `aws.String` and `aws.StringValue` in these cases
so here instead of `&taskArn` use `aws.String(taskArn)`

Reply:
Done...